src/support/action/acceptCookieBanner.js

```diff
@@ -0,0 +1,13 @@
+// To dismiss cookie banner, as it blocks clicks
+
+module.exports = (text) => {
+
+	browser.pause(2000);
```

Comment:
If we're waiting for the cookie banner to exist by pausing, then we should be able to actually wait for a dom element to exist e.g. http://v4.webdriver.io/api/utility/waitForExist.html

Reply:
now using the global-nav element

Reply:
changed to wait on the cookieControl element as some other projects do not make use of the glonalnav.